spec/views/posts/index_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+RSpec.describe 'Post Index', type: :feature do
+  before :each do
+    @avatar_pic = 'https://unsplash.com/photos/F_-0BxGuVvo'
+    @author = User.create(name: 'Cassian Andor',
+                          photo: @avatar_pic,
+                          bio: 'Rebel fighter from Kenari.An accomplished Alliance Intelligence officer
+                          with combat field experience.')
+    @first_post = Post.create(author: @author, title: 'Selam from', text: 'This is my first post')
+    @second_post = Post.create(author: @author, title: 'Hello there!',
+                               text: 'This is ObiWans\' message: Don\'t let the Empire win')
+
+    @com1 = Comment.create(author: @author, post: @second_post, text: 'Thank you for your support 😊')
+    @com2 = Comment.create(author: @author, post: @second_post,
+                           text: 'Hello there Cassian, this Dero I am watching you')
+    @com3 = Comment.create(author: @author, post: @second_post, text: 'Spend the credits wisely :|')
+
+    visit user_posts_path(@author)
+  end
+
+  it 'shows the username of the user' do
+    expect(page).to have_content(@author.name)
+  end
+  it 'shows the user photo' do
+    expect(page.body).to include(@avatar_pic)
+  end
+
+  it 'shows number of posts user has written' do
+    expect(page.body).to include('Number of posts: 2')
+  end
+
+  it 'shows number of comments a post has' do
+    expect(page.body).to include('Comments: 3')
+  end
+
+  it 'shows number of likes a post has' do
+    expect(page.body).to include('Likes: 0')
+  end
+
+  it 'shows the title of the post' do
+    expect(page).to have_content(@second_post.title)
+  end
+
+  it 'shows part of a post body (This is ObiWans\' message: Don\'t let the Empire win)' do
+    expect(page).to have_content(@second_post.text)

```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=I%20can%20see%20the%20first%20comments%20on%20a%20post.), Please add tests that assert you can see the first comments on a post.
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=I%20can%20see%20a%20section%20for%20pagination%20if%20there%20are%20more%20posts%20than%20fit%20on%20the%20view.), Please add Tests that asserts you can see a section for pagination if there are more posts than fit on the view.